doc/source/user_guide/window.rst

```diff
@@ -592,7 +601,7 @@ The following formula is used to compute exponentially weighted mean with an inp
 
 .. math::
 
-    y_t = \frac{\sum_{i=0}^t 0.5^\frac{t_{t} - t_{i}}{\lambda} x_{t-i}}{\sum_{i=0}^t 0.5^\frac{t_{t} - t_{i}}{\lambda}},
```

Comment:
same comment w respect to merging master as above

Reply:
Ok merged

Reply:
if you check the diff, looks unchanged. You might need to fetch first, see https://pandas.pydata.org/docs/development/contributing.html#working-with-the-code for a workflow on updating a pr

Reply:
Okay, I added the changes from master branch file since fetching and merging didn't change those lines..